src/Extensions/ContentReviewDefaultSettings.php

```diff
@@ -130,6 +130,7 @@ public function updateCMSFields(FieldList $fields)
 
         $users = Permission::get_members_by_permission([
             'CMS_ACCESS_CMSMain',
+            'CMS_ACCESS_LeftAndMain',
```

Comment:
Why is this being added? This doesn't look like something that should be covered by unit test and shouldn't be in a MNT commit.

Reply:
Yeah it should probably be in its own bugfix branch